akka-http-core/src/test/scala/akka/http/impl/engine/parsing/HttpHeaderParserSpec.scala

```diff
@@ -150,6 +150,13 @@ class HttpHeaderParserSpec extends WordSpec with Matchers with BeforeAndAfterAll
       parseAndCache("Fancy: foo\tbar\r\nx")() shouldEqual RawHeader("Fancy", "foo bar")
     }
 
+    "parse and cache a header with UTF8 chars in the value" in new TestSetup() {
+      parseAndCache("2-UTF8-Bytes: árvíztűrő ütvefúrógép\r\nx")() shouldEqual RawHeader("2-UTF8-Bytes", "árvíztűrő ütvefúrógép")
```

Comment:
Interesting! What is "flood-resistant chlorophyte drill" in Hungarian, @drewhk? Maybe we can squeeze a few more funky chars in there?